msg/vehicle_status.msg

```diff
@@ -61,7 +66,7 @@ uint8 system_type			# system type, contains mavlink MAV_TYPE
 uint8 system_id			# system id, contains MAVLink's system ID field
 uint8 component_id			# subsystem / component id, contains MAVLink's component ID field
 
-bool is_rotary_wing			# True if system is in rotary wing configuration, so for a VTOL this is only true while flying as a multicopter
```

Comment:
Just to be aware: Flight Review is using this and will require updating - I can do that.

Also can you please keep the original comment about VTOLs?

Reply:
@bkueng would it be better to have both flags for a transition period not to break tools?

Reply:
I'm fine with breaking as long as we don't go back and forth. Flight Review is updated quickly, I just need to do it before this gets merged.

Reply:
I decided to get rid of the `is_rotary_wing` flag completely, because there were some modules that made the assumption that `is_rotary_wing == false` implies that it is a fixed-wing craft. With this new enum value, that mistake is much harder to make in the future, but the same functionality is retained with respect to VTOLs.

Reply:
Flight review is updated: https://github.com/PX4/flight_review/commit/e90161988213bfc3ff4f4827e6594a5623434658